app/src/main/java/com/mapbox/maps/testapp/examples/annotation/SymbolActivity.kt

```diff
@@ -80,10 +66,17 @@ class SymbolActivity : AppCompatActivity() {
         iconAllowOverlap = true
         textAllowOverlap = true
 
+        val airportImage = BitmapUtils.getBitmapFromDrawable(
+          ResourcesCompat.getDrawable(
+            resources,
+            R.drawable.ic_airplanemode_active_black_24dp,
+            this@SymbolActivity.theme
+          )
+        )
         // create a symbol
         val symbolOptions: SymbolOptions = SymbolOptions()
           .withPoint(Point.fromLngLat(AIRPORT_LONGITUDE, AIRPORT_LATITUDE))
-          .withIconImage(ID_ICON_AIRPORT)
+          .withIconImage(airportImage!!)
```

Comment:
Let's not use `!!` here - we could check for null while preparing bitmap above and throw some exception in case `bitmap == null`